corehq/apps/couch_sql_migration/couchsqlmigration.py

```diff
@@ -102,8 +102,8 @@ def setup_logging(log_dir, slug, debug=False):
                 handler.setLevel(logging.DEBUG)
     if not log_dir:
         return
-    time = datetime.utcnow().strftime("%Y%m%d%H%M%S")
-    log_file = os.path.join(log_dir, f"couch2sql-form-case-{time}-{slug}.log")
+    time = datetime.utcnow().strftime("%Y-%m-%d_%H.%M.%S")
+    log_file = os.path.join(log_dir, f"couch2sql_form_case_{time}_{slug}.log")
```

Comment:
Lol. I always format dates in filenames following ISO 8601 because I find them easy to read, but I appreciate that the "_" vs "-" debate is still being hotly contested in my `/etc` directory ("debian_version" vs "lsb-release"). This is such a small issue that instead of editing the code again I think it's easier to leave it as it is.